### ISSUE TITLE
Widget generates error in IE

### ISSUE THREAD

**Issue by royhermans** [mendixlabs/rich-text#28]
Hi Andries,

A user reported an error generated by this widget. After some research it turns out that user is using Internet Explorer. Turns out to be the same issue as reported in issue #23 .

I have updated the widget and now I get a pop-up stating "Failed to sanitize text, please use an other browser
". Is this something that can or will be fixed for Internet Explorer? The problem does not occur in Edge, Chrome, etc.

If you want I can also upload a test project in which I am able to replicate the issue.

Thank you in advance!

Kind regards,

Roy Hermans

**Comment by diego-antonelli** [comment]
Hi @royhermans can you please comment here your html content?

**Comment by diego-antonelli** [comment]
Hi @royhermans, we are facing an issue with % inside URLs just in IE and Edge till version 16. 

Cause: IE and Edge are expecting for URL Entities in <a> elements. So for IE and Edge just % is wrong, 
they are expecting %20 or another URL Entity. Chrome and Firefox just ignore single %.

Suggestion: What I can suggest for you is change the pattern of your replacedVar from {%varName%} to just {varName} and also in your configurations to replace, remove the %. 

You can see more here: https://www.w3schools.com/tags/ref_urlencode.asp

**Comment by royhermans** [comment]
Hi @diego-antonelli , I am not sure we are understanding eachother correctly. I am not using any %'s or even URL's.

I checked a bit more and managed to narrow it down a bit:
The error occurs when viewing the Rich Text Widget **non-editable** in a list-view. 

![screenshot_67](https://user-images.githubusercontent.com/32068390/51895190-db778880-23a9-11e9-90c6-5258cb6db935.png)


When you view the object on a page containing the non-editable Rich Text widget you do not get any errors.

![screenshot_68](https://user-images.githubusercontent.com/32068390/51895203-e500f080-23a9-11e9-8a1d-35beb1a7e5fd.png)

I have also uploaded the test project, just create a new object, enter some random text in the Rich Text Widget and save. The use the Overview Listview page to view the error in Internet Explorer.

[RichTextIE_error.zip](https://github.com/mendixlabs/rich-text/files/2806721/RichTextIE_error.zip)

I hope this clarifies things.
